docs/android/README.md

```diff
@@ -138,121 +255,272 @@ cd pdfapp
-
-```diff
-...
-<platform name="android">
-+   <preference name="AndroidXEnabled" value="true" />
-    <allow-intent href="market:*" />
-</platform>
-...
+5. Add the PDF document you want to display in your project’s `assets` directory. You can use <a href="https://pspdfkit.com/downloads/pspdfkit-android-quickstart-guide.pdf" download="Document.pdf">this QuickStart Guide PDF</a> as an example.
 
+```shell
+cp ~/Downloads/Document.pdf platforms/android/app/src/main/assets/Document.pdf
 ```
 
 6. Now open your `index.js` file located in `www/js/` and paste the below code into the `onDeviceReady()` function. For this to work you need to create a folder called `documents` in `www` and paste a PDF in this folder.
 
 ```javascript
-PSPDFKit.presentFromAssets("www/documents/A.pdf", {
+PSPDFKit.present("file:///android_asset/Document.pdf", {
   title: "My PDF Document",
   page: 0,
   scrollDirection: PSPDFKit.PageScrollDirection.VERTICAL,
   scrollMode: PSPDFKit.ScrollMode.CONTINUOUS,
-  useImmersiveMode: true
+  useImmersiveMode: true,
 });
 ```
 
-7. You are now ready to build and test your app!
+7. [Start your emulator](https://developer.android.com/studio/run/emulator#runningemulator).
+8. The app is now ready to launch:
 
 ```shell
-cordova build
-cordova run
+cordova emulate android
 ```
 
-### Quickstart Guide Ionic
+## Running the PSPDFKit-Demo Cordova Example
 
-For more information regarding the Ionic installation you can check out the [official Ionic installation guide](https://ionicframework.com/docs/v1/guide/installation.html).
+1. Open the Terminal app and clone the GitHub repository:
 
-1. Create a new Ionic project from the command line using the [Ionic Command-Line Interface (CLI)](https://ionicframework.com/docs/cli/start/) .
+```shell
+git clone https://github.com/PSPDFKit/pspdfkit-cordova.git
+```
+
+2. Change the current working directory to the example project’s folder:
 
 ```shell
-ionic start todo blank --type ionic1
+cd pspdfkit-cordova/examples/cordova/PSPDFKit-Demo
 ```
 
-It will then ask you if you want to integrate your new app with Cordova, answer with yes. After the process is finished change to the newly created Ionic project directory.
+2. Install the dependencies:
 
 ```shell
-cd todo
+npm install
 ```
 
-2. Add Android platform support to your project. This plugin requires the latest `cordova-android` plugin 8+.
+3. Add the Android platform:
 
 ```shell
-ionic cordova platform add android
+cordova platform add android
 ```
 
-3. Install the PSPDFKit plugin:
+4. Next, you need to setup your PSPDFKit license key. If you don't have a license key yet, you can get one by requesting an evaluation version of PSPDFKit at https://pspdfkit.com/try. Inside your Android app's `platforms/android/local.properties` file, specify the `pspdfkit.license` property:
+
+```properties
+# This is the LICENSE_KEY you received when requesting a demo or from the customer portal.
+pspdfkit.license=LICENSE_STRING
+```
```

Comment:
This section is misleading for the example. The package ID is already `com.pspdfkit.demo` so any license key that is not `YOUR_LICENSE_KEY_GOES_HERE` will cause the app to crash.

Is it possible to remove this step and replace it by committing a `platforms/android/local.properties` file with the contents:

```properties
pspdfkit.license=YOUR_LICENSE_KEY_GOES_HERE
```

Reply:
Good point! We can't commit the file because the `platforms` directory is not (and should not) be tracked by git. I'll fix this by clarifying this step.